packages/amplify-category-storage/src/provider-utils/awscloudformation/cdk-stack-builder/ddb-stack-transform.ts

```diff
@@ -222,7 +219,9 @@ export class DDBStackTransform {
           },
         });
         try {
-          await sandboxNode.run(overrideCode, overrideJSFilePath).override(this._resourceTemplateObj as AmplifyDDBResourceTemplate);
+          await sandboxNode
+            .run(overrideCode, overrideJSFilePath)
+            .override(this._resourceTemplateObj as AmplifyDDBResourceTemplate);
         } catch (err: $TSAny) {
```

Comment:
should these errors be propagated rather than just printed?

Reply:
yeah, throwing all moving forward